saleor/core/utils/random_data.py

```diff
@@ -900,10 +901,24 @@ def _create_staff_user(staff_password, email=None, superuser=False):
         is_staff=True,
         is_active=True,
         is_superuser=superuser,
+        search_document=_prepare_search_document_value(email, address),
     )
     return staff_user
 
 
+def _prepare_search_document_value(email, address):
```

Comment:
Could we have one function for that?
We have same logic in `saleor/account/search.py::prepare_user_search_document_value`